### DISCUSSION TITLE
Trapdoor and Clemtaminator

### DISCUSSION THREAD

**Discussion by lancer1977** [Pryaxis/TShock#1892] [General]
Saw a few posts over the past years regarding the Clemtaminator biomes reverting back, especially the backgrounds once sprayed (which was the issue I was having). I couldn't find a fix. Then I noticed the trapdoor would break when used also. Someone had mentioned a solution for it, and it worked on the clemtaminator issue for me as well. Have to add
tshock.ignore.sendtilesquare
permission to default group.

**Comment by hakusaro** [comment]
Does ` tshock.ignore.sendtilesquare` fix trap doors for you? Other people have reported that it _doesn't_ solve the problem for them.

**Comment by lancer1977** [comment]
So I'll include my config as I was trying to adjust some things to enable the clemtaminator (sp) initially. But yes, the trapdoor was not working, I added the tshock.ignore.sendtilesquare to the group and it began working. I may have had to log out and back in, but in either case I decided to retest it tonight and tried a group without the permission and found it "broken" again. After adding the permission and logging out and back in (it didn't seem to recognize the group change or it may have been user error there), it was working properly again.

> `{
  "InvasionMultiplier": 1,
  "DefaultMaximumSpawns": 5,
  "DefaultSpawnRate": 600,
  "ServerPort": 7779,
  "EnableWhitelist": false,
  "InfiniteInvasion": false,
  "PvPMode": "normal",
  "SpawnProtection": true,
  "SpawnProtectionRadius": 10,
  "MaxSlots": 16,
  "RangeChecks": true,
  "DisableBuild": false,
  "SuperAdminChatRGB": [
    255,
    255,
    255
  ],
  "SuperAdminChatPrefix": "(Super Admin) ",
  "SuperAdminChatSuffix": "",
  "BackupInterval": 0,
  "BackupKeepFor": 60,
  "RememberLeavePos": false,
  "HardcoreOnly": false,
  "MediumcoreOnly": false,
  "KickOnMediumcoreDeath": false,
  "BanOnMediumcoreDeath": false,
  "AutoSave": true,
  "AnnounceSave": true,
  "MaximumLoginAttempts": 3,
  "ServerName": "",
  "UseServerName": false,
  "StorageType": "sqlite",
  "SqliteDBPath": "tshock.sqlite",
  "MySqlHost": "localhost:3306",
  "MySqlDbName": "",
  "MySqlUsername": "",
  "MySqlPassword": "",
  "MediumcoreBanReason": "Death results in a ban",
  "MediumcoreKickReason": "Death results in a kick",
  "EnableIPBans": true,
  "EnableUUIDBans": true,
  "EnableBanOnUsernames": false,
  "DefaultRegistrationGroupName": "default",
  "DefaultGuestGroupName": "guest",
  "DisableSpewLogs": true,
  "DisableSecondUpdateLogs": false,
  "HashAlgorithm": "sha512",
  "ServerFullReason": "Server is full",
  "WhitelistKickReason": "You are not on the whitelist.",
  "ServerFullNoReservedReason": "Server is full. No reserved slots open.",
  "SaveWorldOnCrash": true,
  "EnableGeoIP": false,
  "EnableTokenEndpointAuthentication": false,
  "RestApiEnabled": false,
  "RestApiPort": 7878,
  "DisableTombstones": true,
  "DisplayIPToAdmins": false,
  "KickProxyUsers": true,
  "DisableHardmode": false,
  "DisableDungeonGuardian": false,
  "DisableClownBombs": false,
  "DisableSnowBalls": false,
  "ChatFormat": "{1}{2}{3}: {4}",
  "ChatAboveHeadsFormat": "{2}",
  "ForceTime": "normal",
  "TileKillThreshold": 300,
  "TilePlaceThreshold": 300,
  "TileLiquidThreshold": 300,
  "ProjectileThreshold": 150,
  "HealOtherThreshold": 50,
  "ProjIgnoreShrapnel": true,
  "RequireLogin": false,
  "DisableInvisPvP": false,
  "MaxRangeForDisabled": 10,
  "ServerPassword": "",
  "RegionProtectChests": false,
  "RegionProtectGemLocks": true,
  "DisableLoginBeforeJoin": false,
  "DisableUUIDLogin": false,
  "KickEmptyUUID": false,
  "AllowRegisterAnyUsername": false,
  "AllowLoginAnyUsername": true,
  "MaxDamage": 1175,
  "MaxProjDamage": 1175,
  "KickOnDamageThresholdBroken": false,
  "IgnoreProjUpdate": false,
  "IgnoreProjKill": false,
  "AllowIce": false,
  "AllowCrimsonCreep": false,
  "AllowCorruptionCreep": false,
  "AllowHallowCreep": false,
  "StatueSpawn200": 3,
  "StatueSpawn600": 6,
  "StatueSpawnWorld": 10,
  "PreventBannedItemSpawn": false,
  "PreventDeadModification": true,
  "EnableChatAboveHeads": false,
  "ForceXmas": false,
  "AllowAllowedGroupsToSpawnBannedItems": false,
  "IgnoreChestStacksOnLoad": false,
  "LogPath": "tshock",
  "UseSqlLogs": true,
  "RevertToTextLogsOnSqlFailures": 10,
  "PreventInvalidPlaceStyle": true,
  "BroadcastRGB": [
    127,
    255,
    212
  ],
  "ApplicationRestTokens": {},
  "ReservedSlots": 20,
  "LogRest": false,
  "RespawnSeconds": 5,
  "RespawnBossSeconds": 10,
  "TilePaintThreshold": 15,
  "ForceHalloween": false,
  "AllowCutTilesAndBreakables": false,
  "CommandSpecifier": "/",
  "CommandSilentSpecifier": ".",
  "KickOnHardcoreDeath": false,
  "BanOnHardcoreDeath": false,
  "HardcoreBanReason": "Death results in a ban",
  "HardcoreKickReason": "Death results in a kick",
  "AnonymousBossInvasions": true,
  "MaxHP": 600,
  "MaxMP": 500,
  "SaveWorldOnLastPlayerExit": true,
  "BCryptWorkFactor": 7,
  "MinimumPasswordLength": 4,
  "RESTMaximumRequestsPerInterval": 5,
  "RESTRequestBucketDecreaseIntervalMinutes": 1,
  "ShowBackupAutosaveMessages": true,
  "DebugLogs": false
}`

**Comment by hakusaro** [comment]
> I added the tshock.ignore.sendtilesquare to the group and it began working

This is a known issue -- we're working on fixing it and you can track progress and watch https://github.com/Pryaxis/TShock/issues/1774 for progress on this.
